src/main/java/com/zufar/onlinestore/payment/exception/PaymentEventProcessingException.java

```diff
@@ -0,0 +1,14 @@
+package com.zufar.onlinestore.payment.exception;
+
+import lombok.Getter;
+
+@Getter
+public class PaymentEventProcessingException extends RuntimeException {
+
+    private final String stripeSignature;
+
+    public PaymentEventProcessingException(String stripeSignature) {
+        super(String.format("Payment event with the stripe signature '%s' cannot be processed.", stripeSignature));
```

Comment:
"Payment event with the stripe signature = '%s' cannot be processed."

Reply:
Done